cloud/amazon/cloudformation_facts.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: cloudformation_facts
+short_description: Obtain facts about an AWS CloudFormation stack
+description:
+  - Gets information about an AWS CloudFormation stack
+dependencies:
+  - boto3>=1.0.0
```

Comment:
This documentation field needs to be requirements, not dependencies.

I think that boto3 requires python >= 2.6?  If so, need to mention python >= 2.6 as a requirement as well.